src/cdp-proxy/reactNativeCDPProxy.ts

```diff
@@ -31,11 +23,24 @@ export class ReactNativeCDPProxy {
         APPLICATION_REPLY: "Reply From Target To Debugger",
     };
 
+    private server: Server | null;
+    private hostAddress: string;
+    private port: number;
+    private debuggerTarget: Connection;
+    private applicationTarget: Connection;
+    private logger: OutputChannelLogger;
+    private logLevel: LogLevel;
+    private firstStop: boolean;
+    private dbgEndpointHelper: DebuggerEndpointHelper;
```

Comment:
Why not just `debugEnpointHelper`?

Reply:
`debugEnpointHelper` consists of 22 letters. It seems to be too long name, so I decided to shorten it a bit.

Reply:
Not sure if this is the right place to shorten. Other variables are written without any shortenings.